Index: Client/obj/Debug/net5.0/Client.GeneratedMSBuildEditorConfig.editorconfig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>is_global = true\r\nbuild_property.TargetFramework = net5.0\r\nbuild_property.TargetPlatformMinVersion = \r\nbuild_property.UsingMicrosoftNETSdkWeb = true\r\nbuild_property.ProjectTypeGuids = \r\nbuild_property.PublishSingleFile = \r\nbuild_property.IncludeAllContentForSelfExtract = \r\nbuild_property._SupportedPlatformList = Android,iOS,Linux,macOS,Windows\r\nbuild_property.RootNamespace = Client\r\nbuild_property.ProjectDir = C:\\Users\\Vosta\\OneDrive\\Documents\\GitHub\\Small-paws-project\\Small paws\\Client\\\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/obj/Debug/net5.0/Client.GeneratedMSBuildEditorConfig.editorconfig b/Client/obj/Debug/net5.0/Client.GeneratedMSBuildEditorConfig.editorconfig
--- a/Client/obj/Debug/net5.0/Client.GeneratedMSBuildEditorConfig.editorconfig	(revision 656e4f6850b6f43f4ca178b79fb907ee09b20190)
+++ b/Client/obj/Debug/net5.0/Client.GeneratedMSBuildEditorConfig.editorconfig	(date 1635512035779)
@@ -7,4 +7,4 @@
 build_property.IncludeAllContentForSelfExtract = 
 build_property._SupportedPlatformList = Android,iOS,Linux,macOS,Windows
 build_property.RootNamespace = Client
-build_property.ProjectDir = C:\Users\Vosta\OneDrive\Documents\GitHub\Small-paws-project\Small paws\Client\
+build_property.ProjectDir = C:\Users\hansl\Documents\GitHub\Small-paws-project\Small paws\Client\
Index: Client/obj/rider.project.restore.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>16354243913442484
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Client/obj/rider.project.restore.info b/Client/obj/rider.project.restore.info
--- a/Client/obj/rider.project.restore.info	(revision 656e4f6850b6f43f4ca178b79fb907ee09b20190)
+++ b/Client/obj/rider.project.restore.info	(date 1635512035800)
@@ -1,1 +1,1 @@
-16354243913442484
\ No newline at end of file
+16354331390282480
\ No newline at end of file
